config/samples/dataplane_v1beta1_openstackdataplanenodeset_networker.yaml

```diff
@@ -18,16 +18,16 @@ spec:
     - telemetry
```

Comment:
now that you are reshaping this... I think the telemetry is related to nova/libvirt and can be removed too